src/testUtils.ts

```diff
@@ -123,6 +123,11 @@ export function goTest(testconfig: TestConfig): Thenable<boolean> {
 			return Promise.resolve();
 		}
 
+		// append the package name to args if applicable
```

Comment:
- this might not work in case of multiple GOPATHs, because then `process.env['GOPATH']` can have `:` or `;` separated paths. Follow https://github.com/Microsoft/vscode-go/blob/0.6.63/src/goCheck.ts#L180-L181 instead
- we are moving away from using GOPATH from `process.env`. Instead use `getCurrentGoPath()`
- When the command `Go: Test All Packages In Workspace` is run, this `go test` command will fail with "can't load package.. no buildable Go source files in ..." if the workspace itself is just a folder containing packages and not a package itself

Reply:
PTAL again, I am not sure how the third item can be solved(and seems it is more than the effort here)

Reply:
`testconfig.includeSubDirectories` is `true` only for the "test all pks in workspace" scenario.
So you can add a check to ensure `testconfig.includeSubDirectories` is `false` before adding the importPath